### ISSUE TITLE
Wrong GitLab releases url

### ISSUE THREAD

**Issue by eMarek** [release-it/release-it#665]
When new release is done, the feedback url is wrong. I am using GitLab. Here is a screenshot:

<img width="405" alt="Screenshot 2020-05-13 at 22 53 37" src="https://user-images.githubusercontent.com/311634/81864497-040a9e00-956d-11ea-9f1a-461697c65da4.png">

This one is written out and leads to 404:
`https://gitlab.com/[group]/[subgroup]/[project]/releases`

This one is correct:
`https://gitlab.com/[group]/[subgroup]/[project]/-/releases`

Spot the missing dash in the path.

By the way is it maybe possible to print any other link at the end? I would love to have pipelines and container registry:
`https://gitlab.com/[group]/[subgroup]/[project]/-/pipelines`
`https://gitlab.com/[group]/[subgroup]/[project]/container_registry`

**Comment by webpro** [comment]
Thanks for reporting this! Just released a fix with [v13.6.0](https://github.com/release-it/release-it/releases/tag/13.6.0).

You can print such messages like this:

```json
{
  "hooks": {
    "after:release": "echo https://gitlab.com/${repo.owner}/[subgroup]/${repo.repository}/-/pipelines"
  }
}
```

Please start one run with `DEBUG=release-it:* release-it [...]` and you should find some corresponding debug output like this (as I'm not sure how to deal with the `[subgroup]` part):

```
  release-it:config { latestTag: '13.6.0' } +554ms
  release-it:config {
  release-it:config   repo: {
  release-it:config     host: 'github.com',
  release-it:config     owner: 'release-it',
  release-it:config     project: 'release-it',
  release-it:config     protocol: 'ssh',
  release-it:config     remote: 'git@github.com:release-it/release-it.git',
  release-it:config     repository: 'release-it/release-it'
  release-it:config   }
  release-it:config }
```

**Comment by webpro** [comment]
Oh, and you can use arrays, like this:

```json
{
  "hooks": {
    "after:release": ["echo a", "echo b"]
  }
}
```
